add link to supplement figs (#19)
greenelab/iscb-diversity-manuscript: content/10.methods.md

For example, a query to <https://genderize.io> on January 26, 2020 for "Casey" returns a probability of male of .74 and a probability of female of 0.26, which we would add for an author with this first name.
Because of the limitations of considering gender as a binary trait and inferring it from first names, we only consider predictions in aggregate and not as individual values for specific scientists.

Excluding 45 authors with empty fore name field, genderize.io fails to provide gender predictions for 3,044 fore names of 34,005 corresponding authors, and two names of 411 ISCB honorees.

> Of these names, approximately 45% have no more than three characters which are likely initials (e.g., "AB", "A.B", or "A B"), and 27% have a hyphen and more than three characters.

Reviewer: I think this description is different than our processing pipeline. A name like "AB" would be set to None in the simplify_fore_name step. Therefore, we never make a genderize call for the name. I imagine the following categories:

1. author does not have fore name (i.e. missing metadata)
2. author has fore name that is None once simplified (i.e. initials)
3. author has fore name that genderize cannot predict
4. author has fore name that genderize can predict

We should differentiate 2 and 3 (but it may be okay to group 1 and 2 together).

For example, a query to <https://genderize.io> on January 26, 2020 for "Casey" returns a probability of male of .74 and a probability of female of 0.26, which we would add for an author with this first name.
Because of the limitations of considering gender as a binary trait and inferring it from first names, we only consider predictions in aggregate and not as individual values for specific scientists.

Of 411 ISCB honorees, genderize.io fails to provide gender predictions for two names.

> Of 34,005 corresponding authors, 45 have empty fore name field (i.e., missing metadata), 1,466 have fore name that is None once simplified (i.e., initials), and 1,578 have fore name for which genderize.io did not provide predictions.

Reviewer: ```suggestion
Of 34,005 corresponding authors, 45 were missing a fore name altogether in the raw paper metadata and 1,466 had fore names consisting only initials.
Of the remaining authors, genderize.io failed to predict gender for 1,578 of these fore names.
```

> We note that approximately 52% of these NA predictions are hyphenated names, which is likely because they are more unique and thus are more difficult to find predictions for.

Reviewer: ```suggestion
52% of the names where genderize.io failed include hyphens, presumably since hyphenated names are more unique and therefore not represented in their training data.
```

We modified the function to return a status denoting that results were inconclusive instead.
This prediction represents the probability of an honoree or author selecting a certain race or ethnicity on a census form if they lived within the United States.

Of 411 ISCB honorees, `wru` fails to provide race/ethnicity predictions for 98 names.

> Of 34,050 corresponding authors, 40 have empty last name field (i.e., missing metadata), and 8,770 have fore name for which `wru` did not provide predictions.

Reviewer: ```suggestion
Of 34,050 corresponding authors, 40 were missing a last name in the paper metadata, and 8,770 had a last name for which `wru` did not provide predictions.
```

> In addition to NA predictions, another limitation of not only `wru` but other race/ethnicity and nationality prediction methods can be inaccurate prediction for women scientists who changed their last name to their husband's if they have different race/ethnicity.

Reviewer: ```suggestion
One limitation of `wru` and other methods that infer race, ethnicity, or nationality from last names is potentially inaccurate prediction for scientists who changed their last name during marriage, a practice more common among women than men.
```
